README.md

```diff
@@ -6,56 +6,49 @@
 
 ![logo](https://raw.githubusercontent.com/opentok/accelerator-core/master/tokbox-logo.png)
 
-# OpenTok Accelerator Core
+# OpenTok Accelerator Core JS
 
+OpenTok's `Accelerator Core JS` provides easy integration for OpenTok libraries. Whether you've already built your application using **OpenTok's libraries** or are just starting to use them, `Accelerator Core JS` helps you implement into your application:
```

Comment:
Whether you've already built your application using the OpenTok platform or are just getting started, . . .